Amazon.IonHashDotnet.Tests/Amazon.IonHashDotnet.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net48</TargetFrameworks>
```

Comment:
I just did this on #47 and now I realize. Do we need to update the same line in the src solution? 
https://github.com/amzn/ion-hash-dotnet/blob/597ec2eae3cf9308daa55fa123517e37d1d18f14/Amazon.IonHashDotnet/Amazon.IonHashDotnet.csproj#L4

Reply:
No, we don't need to. Net Standard is the generic standard which is the ideal target for a library like IOn and IOn hash.

Reply:
I see. Do we need to add `netstandard2.1` though? https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Reply:
yes, that would be a good idea. 2.1 is the LTS version while 2.0 has stopped support.

Reply:
> yes, that would be a good idea. Net Standard 2.1 is the new standard and correspond to .net core 3.0/3.1.

Could I ask you to make this change as well :) Then we can merge this PR and call it done!

Reply:
I looked a bit more into this and I was wrong about .Net Standard 2.1. It is not supported by any version of .Net Framework at all at the moment... So we cannot really use it.